### PR TITLE
added InvokeMethod on WmiClass

### PR DESCRIPTION
**Pull request by jocardeMSFT**
Adding ability to call static methods on Wmi Classes using literal parameters in line with pattern of Wmi Instances

### CODE REVIEW

File: pkg/wmiinstance/WmiClass.go [hunk 1]
```diff
@@ -285,6 +285,16 @@ func (c *WmiClass) InvokeMethod(methodName string, methodParams []string, inputO
 	// both are SWbemObjects: https://docs.microsoft.com/en-us/windows/win32/wmisdk/swbemobject
 }
 
+func (c *WmiClass) InvokeMethodWithLiteralParams(methodName string, params ...interface{}) ([]interface{}, error) {
```
Comment (madhanrm) [inline pkg/wmiinstance/WmiClass.go:288]: Where are we planning to use this?

Comment (jocardeMSFT) [reply pkg/wmiinstance/WmiClass.go:288]: We use it to create cluster group sets, before we didn't have a way to invoke static methods on wmi classes.

https://github.com/jocardeMSFT/moc-pkg/blob/13ffa1b226223a15f1eff1dfa64bb64695eff408/pkg/failovercluster/fc_avset_windows.go#L90C2-L90C183

Comment (JocelynBerrendonner) [reply pkg/wmiinstance/WmiClass.go:288]: Also, how different is this from the unimplemented InvokeMethod call that is right above?

Comment (JocelynBerrendonner) [reply pkg/wmiinstance/WmiClass.go:288]: In other words, let's just name this function "InvokeMethod"

Comment (jocardeMSFT) [reply pkg/wmiinstance/WmiClass.go:288]: the method above takes a methodParams []string argument, this definition takes effectively an array of interfaces.

(one of the params on the call we invoke this is actually a string[] itself.

Comment (jocardeMSFT) [reply pkg/wmiinstance/WmiClass.go]: replaced call, let me know if this was your intent @JocelynBerrendonner

**Review by JocelynBerrendonner — APPROVED** [review]
LGTM

File: pkg/wmiinstance/WmiClass.go (hunk 1 above)
Comment (JocelynBerrendonner) [reply pkg/wmiinstance/WmiClass.go]: Thanks! Looks good.

**Review by madhanrm — APPROVED** [review]

**Pull request merged**